bokehjs/src/lib/models/widgets/color_picker.ts

```diff
@@ -0,0 +1,83 @@
+import {logger} from "core/logging"
+import * as p from "core/properties"
+import {empty, label, input} from 'core/dom'
+import {InputWidget, InputWidgetView} from 'models/widgets/input_widget'
+import {Color} from 'core/types'
+
+
+export class ColorPickerView extends InputWidgetView {
+  model: ColorPicker
+
+  protected inputEl: HTMLInputElement
+
+  initialize(options: any): void {
+    super.initialize(options)
+    this.render()
+  }
+
+  connect_signals(): void {
+    super.connect_signals()
+    this.connect(this.model.change, () => this.render())
+  }
+
+  render(): void {
+    super.render()
+
+    empty(this.el)
+
+    const labelEl = label({for: this.model.id}, this.model.title)
+    this.el.appendChild(labelEl)
+
+    this.inputEl = input({
+      class: "bk-widget-form-input",
+      id: this.model.id,
+      name: this.model.name,
+      value: this.model.color,
+      disabled: this.model.disabled,
+      type: "color",
```

Comment:
This isn't supported in IE and possibly Safari (sources vary).

Reply:
It would be nice if there was a simple fallback that could be supported by those browsers.

Reply:
Ok but is it ok for the spinner?

Reply:
Yeah, I think that `type="number"` is supported well enough. On a side note, going through other types, it's interesting to me that `type="range"` is so well supported.